File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/models/chat_completion_list.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class ChatCompletionList:
    """OpenAI标准聊天完成列表"""
    object: str = None
    data: List[ChatCompletionResponse] = None
    has_more: bool = None
    first_id: str = None
    last_id: str = None

File: sdkwork-ai-sdk-typescript/src/types/chat-completion-list.ts
import type { ChatCompletionResponse } from './chat-completion-response';

/** OpenAI标准聊天完成列表 */
export interface ChatCompletionList {
  /** 对象类型 */
  object?: string;
  /** 聊天完成列表 */
  data?: ChatCompletionResponse[];
  /** 是否有更多数据 */
  has_more?: boolean;
  /** 第一条记录ID */
  first_id?: string;
  /** 最后一条记录ID */
  last_id?: string;
}
